src/server/pfs/fuse/filesystem.go

```diff
@@ -333,11 +333,13 @@ func (f *file) delimiter() pfsclient.Delimiter {
 }
 
 func (f *file) touch() error {
```

Comment:
Yes, which is arguably pretty weird. I think we'll just not use split through fuse.